// @payvo/sdk-ark/distribution/cjs/networks/tsxp.testnet.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const shared_1 = require("./shared");
const network = {
    coin: "Solar Testnet",
    constants: {
        slip44: 1,
    },
    currency: {
        decimals: 8,
        symbol: "tSXP",
        ticker: "tSXP",
    },
    explorer: {
        block: "block/{0}",
        transaction: "transactions/{0}",
        wallet: "wallet/{0}",
    },
    featureFlags: {
        ...shared_1.featureFlags,
        Transaction: [
            "delegateRegistration",
            "delegateResignation",
            "estimateExpiration",
            "multiPayment",
            "secondSignature",
            "transfer",
            "vote",
        ],
    },
    governance: {
        delegateCount: 53,
        method: "split",
        votesPerTransaction: 1,
        votesPerWallet: 1,
    },
    hosts: [{
            host: "https://sxp1.testnet.sh/api",
            type: "full",
        },
        {
            host: "https://texplorer.solar.org",
            type: "explorer",
        },
    ],
    id: "tsxp.testnet",
    importMethods: shared_1.importMethods,
    knownWallets: "https://raw.githubusercontent.com/solar-network/common/master/testnet/known-wallets-extended.json",
    meta: {
        fastDelegateSync: true,
    },
    name: "Testnet",
    transactions: {
        ...shared_1.transactions,
        fees: {
            ticker: "tSXP",
            type: "dynamic",
        },
        multiPaymentRecipients: 128,
    },
    type: "live",
};
exports.default = network;
