// @payvo/sdk-ark/distribution/esm/networks/tsxp.testnet.js
import { featureFlags, importMethods, transactions } from "./shared";
const network = {
    coin: "Solar",
    constants: {
        slip44: 1,
    },
    currency: {
        decimals: 8,
        symbol: "tSXP",
        ticker: "tSXP",
    },
    explorer: {
        block: "testnet/block/{0}",
        transaction: "testnet/transactions/{0}",
        wallet: "testnet/wallet/{0}",
    },
    featureFlags: {
        ...featureFlags,
        Transaction: [
            "delegateRegistration",
            "delegateResignation",
            "estimateExpiration",
            "multiPayment",
            "secondSignature",
            "transfer",
            "vote",
        ],
    },
    governance: {
        delegateCount: 53,
        method: "split",
        votesPerTransaction: 1,
        votesPerWallet: 1,
    },
    hosts: [{
            host: "https://sxp.testnet.sh/api",
            type: "full",
        },
        {
            host: "https://explorer.solar.org",
            type: "explorer",
        },
    ],
    id: "tsxp.testnet",
    importMethods,
    meta: {
        fastDelegateSync: true,
    },
    name: "Testnet",
    transactions: {
        ...transactions,
        fees: {
            ticker: "tSXP",
            type: "dynamic",
        },
        multiPaymentRecipients: 128,
    },
    type: "test",
};
export default network;
